test/release/examples/primers/modulesHelper.chpl

```diff
@@ -0,0 +1,13 @@
+/* This file is solely intended as a helper file to the modules primer.  It
+   is not intended to be looked at on its own.
+*/
```

Comment:
Another advantage to removing the material related to files and directories is that it would avoid having to inject the extra files and directories to support this primer...